apis/filters/parse_test.go

```diff
@@ -223,3 +223,53 @@ func TestArgsMatchKVList(t *testing.T) {
 		}
 	}
 }
+func TestArgsValidate(t *testing.T) {
```

Comment:
Please add a blank line before this line. There should be a blank line in every function. @WillSmisi

Reply:
Got it!

Reply:
> Please add a blank line before this line. There should be a blank line in every function. @WillSmisi

Dear @allencloud, Thank you for telling me the message, I 'm a green hand. If there is any improvement that I can do, please let me know. 
Thank you with my appreciation!